src/util/sound_api_util.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const getSoundById = (id) => {
+  return axios.get("/v1/me/sounds/" + id);
+};
+
+export const getSoundByIds = (ids) => {
+  debugger
+  const params = {
+    "ids": ids
+  }
+  return axios.get("/v1/me/sounds?ids=5fcadb14d62a7b2f68b3cb27,5fcadf1dd62a7b2f68b3cb28");
```

Comment:
For some reason, when I pass ids (array), the URL comes out to be like **?ids[]=id1,id2** which is incorrect format. Need to figure out how to do it correctly. Might need to iterate array and append to a string instead.